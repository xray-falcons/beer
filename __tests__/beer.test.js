import React from 'react';
import Beer from "../components/beer";
import 'react-native';

import renderer from 'react-test-renderer';

describe('<Beer />', () => {
    let beer = {
        name: "Ipa",
        labels: {
            medium: "URL"
        }
    }
    it('renders correctly', () => {

        const tree = renderer.create(
            <Beer beer={beer}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});



// import React from 'react';
// import App from './App';
//
// import 'react-native';
//
// import renderer from 'react-test-renderer';
// jest.mock('NativeAnimatedHelper');
//
// describe('<App />', () => {
//     it('renders correctly', () => {
//         const navigation = { navigate: jest.fn() };
//
//         const tree = renderer.create(
//             <App navigation={navigation}/>
//         ).toJSON();
//         expect(tree).toMatchSnapshot();
//     });
// });
