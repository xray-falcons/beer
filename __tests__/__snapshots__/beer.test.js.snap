// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Beer /> renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "URL",
        }
      }
      style={
        Object {
          "height": 100,
          "resizeMode": "cover",
          "width": 100,
        }
      }
    />
    <Text>
      Ipa
    </Text>
  </View>
</View>
`;
